odl/tomo/geometry/conebeam.py

```diff
@@ -847,9 +890,34 @@ def angles(self):
         """Discrete angles given in this geometry."""
         return self.motion_grid.coord_vectors[0]
 
-    def det_axes(self, angles):
-        """Return the detector axes tuple at ``angle``."""
-        return tuple(self.rotation_matrix(angles).dot(axis)
+    def det_axes(self, angle):
+        """Return the detector axes tuple at ``angle``.
+
+        Parameters
+        ----------
+        angles : float or `array-like`
+            Angle(s) in radians describing the counter-clockwise rotation
+            of the detector around `axis`.
+
+        Returns
+        -------
+        axes : tuple of `numpy.ndarray`'s
```

Comment:
Do we want the same for the `axes` property of detectors? Single array instead of tuple?